app/models/concerns/annotation_data.rb

```diff
@@ -0,0 +1,85 @@
+module AnnotationData
+  include JsonReader
+  extend ActiveSupport::Concern
+
+  #attr_accessor :annotation_url, :annotation_list
+
+  def read_annotation(url=nil)
+    return nil unless url #self[:annotation_url]
+    @annotation_list = JsonReader::Reader.new.from_url(url)
+    #self[:annotation_list] = JsonReader::Reader.new.from_str(self[:annotation_url])
+  end
+
+  def motivation_for_annotations
+    "oa:commenting"
+  end
+
+  def motivation_for_transcriptions
+    "sc:painting"
+  end
+
+  def annotations(annotation_list=nil)
+    # the motivation for annotations will be: "oa:commenting"
+    #return [] unless self[:annotation_url]
+    #self.read_annotation unless self[:annotation_list]
+    #return [] unless self[:annotation_list]
+    #return self[:annotation_list][:resources].select {|anno| anno["motivation"] == "oa:commenting" }
+    return [] unless annotation_list
+    return annotation_list["resources"].select {|anno| anno["motivation"] == self.motivation_for_annotations }
+  end
+
+  def transcriptions(annotation_list=nil)
+    # the motivation for transcriptions will be: "sc:painting"
+    #return [] unless self[:annotation_url]
+    #self.read_annotation unless self[:annotation_list]
+    #return [] unless self[:annotation_list]
+    #return self[:annotation_list][:resources].select {|anno| anno["motivation"] == "sc:painting" }
+    return [] unless annotation_list
+    return annotation_list["resources"].select {|anno| anno["motivation"] == self.motivation_for_transcriptions }
+  end
+
+  def map_annotation(annotation=nil)
+    return {} unless annotation
+    anno = {}
+    anno['id'] = annotation["@id"]
```

Comment:
Seems like there is a mix of single quotes and double quotes. Would be good to standardize for the project. I normally defer to [ruby style guide](https://github.com/bbatsov/ruby-style-guide)